### PR TITLE
WIP: Generate message signalling chart

### PR DESCRIPTION
**Pull request by stappersg**
new file: docs/screenshots/Makefile
new file: docs/screenshots/shoelaces-overview.msc

To make it possible to generate the docs/screenshots/shoelaces-overview.png

### CODE REVIEW

**Comment by stappersg** [comment]
Binary file for in a separate merge request.
shoelaces-overview

**Comment by stappersg** [comment]
This merge request is _work in progress_.  Feedback/comments/rejects are welcome, but please do not yet merge.

File: docs/screenshots/Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+#
+shoelaces-overview.png: shoelaces-overview.msc
+	msc-gen \
+		-T png \
+		-i shoelaces-overview.msc \
+		-o shoelaces-overview.png
+
+#
+# FYI  mscgen homepage is https://gitlab.com/msc-generator/msc-generator
+# l l
```
Comment (raul-te) [inline docs/screenshots/Makefile:12]: Typo?

File: docs/screenshots/shoelaces-overview.msc [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  [line.corner=round]
+  {
+
+    "Booting System"->Shoelaces:HTTP get\n/poll/1/m-ac-a-dd-re-ss;
+    Shoelaces->"Booting System":HTTP response\nNext instructions;
+  };
+
+"Booting System"->"Booting System":Executes\ninstructions;
+
+# l l
```
Comment (raul-te) [inline docs/screenshots/shoelaces-overview.msc:31]: Extra line and typo? Not sure what's `# l l`, but it was on the other file as well.

**Review by raul-te — COMMENTED** [review]
This contribution will be most welcome. Thank you.

File: docs/screenshots/Makefile (hunk 1 above)
Comment (stappersg) [reply docs/screenshots/Makefile:12]: `# l l`: a comment to express  _Last Line_.

File: docs/screenshots/shoelaces-overview.msc (hunk 2 above)
Comment (stappersg) [reply docs/screenshots/shoelaces-overview.msc:31]: `# l l`: a comment to express  _Last Line_.

No, not a typo. an _End of File_ is a good thing when there is no closing tag, e.g. `</html>`, nor a closing curly bracket (as in `json`, `C`, `rust` or `go`).

**Comment by stappersg** [comment]
Closed in favour of #29.  (And the _binary change_ did happen in #28.)

**Comment by stappersg** [comment]
Closed in favour of #29.  (And the _binary change_ did happen in #28.)

**Pull request closed without merging**